src/action-creators/moveThoughtDown.js

```diff
@@ -30,8 +30,14 @@ export const moveThoughtDown = () => dispatch => {
       // metaprogramming functions that prevent moving
       const thoughtMeta = meta(pathToContext(cursor))
       const contextMeta = meta(pathToContext(contextOf(cursor)))
+      const globalSort = localStorage['Settings/Global Sort'] || 'None'
```

Comment:
I recommend [`getSetting(['Global Sort'])`](https://github.com/cybersemics/em/blob/dev/src/util/getSetting.js) here. `localStorage` is used as a persistence layer from which `state` is loaded at startup. The Redux store will be faster for in-app lookups.